torchtext/experimental/transforms.py

```diff
@@ -1,483 +1,61 @@
-        mask_bos (bool): indicate whether beginning-of-sequence tokens are eligible for masking (default: False)
-        mask_prob (float): probability that a token is chosen for replacement (default: 0.15)
-
-    Example:
-        >>> import torch
-        >>> from torchtext.experimental.transforms import MaskTransform
-        >>> sample_tokens = [
-                ["[BOS]", "a", "b", "c", "d"],
-                ["[BOS]", "a", "b", "[PAD]", "[PAD]"]
-            ]
-        >>> sample_token_ids = torch.tensor([
-                [6, 0, 1, 2, 3], [6, 0, 1, 4, 4]
-            ])
-        >>> mask_transform = MaskTransform(
-                vocab_len = 7,
-                mask_idx = 4,
-                bos_idx = 6,
-                pad_idx = 5,
-                mask_bos = False,
-                mask_prob = 0.15
-            )
-        >>> masked_tokens, target_tokens, mask = mask_transform(sample_token_ids)
-    """
-
-    # maks_mask_prob is prob. of replacing a token with [MASK] (ex. 80%)
-    mask_mask_prob = 0.8
-
-    # rand_mask_thresh is prob. of replacing a token with a random token. (ex.10%)
-    rand_mask_prob = 0.1
-
-    def __init__(
-        self,
-        vocab_len: int,
-        mask_idx: int,
-        bos_idx: int,
-        pad_idx: int,
-        mask_bos: bool = False,
-        mask_prob: float = 0.15,
-    ):
-        super().__init__()
-        self.vocab_len = vocab_len
-        self.mask_idx = mask_idx
-        self.bos_idx = bos_idx
-        self.pad_idx = pad_idx
-        self.mask_prob = mask_prob
-        self.mask_bos = mask_bos
-
-    def forward(self, tokens: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor]:
-        """
-        Applies mask to input tokens.
-
-        Inputs:
-            tokens: Tensor with token ids of shape (batch_size x seq_len). Includes token ids for special tokens such as [BOS] and [PAD]
-
-        Outputs:
-            masked_tokens: Tensor of tokens after masking has been applied
-            target_tokens: Tensor of token values selected for masking
-            mask: Tensor with same shape as input tokens (batch_size x seq_len)
-                with masked tokens represented by a 1 and everything else as 0.
-        """
-        # tokens, mask, mask_mask, rand_mask: (T, C)
-        mask, mask_mask, rand_mask = self._generate_mask(tokens)
+    if name in moved_apis:
+        import warnings
 
-        # a. generate the masked input tokens
-        # (1) the [MASK] token 80% of the time
-        masked_tokens = self._mask_input(tokens, mask_mask, self.mask_idx)
-        # (2) a random token 10% of the time
-        masked_tokens = self._mask_input(
-            masked_tokens,
-            rand_mask,
-            torch.randint_like(tokens, high=self.vocab_len),
+        warnings.warn(
+            "experimental package has been moved to prototype. You may change all imports from `experimental` to `prototype`",
+            DeprecationWarning,
```

Comment:
```suggestion
            UserWarning,
```

`DeprecationWarning` is default not-shown, so I think `UserWarning` is better. (yes, I know DeprecationWarning is used in the example I cited, which I should fix. :) )